src/main/kotlin/net/atos/zac/smartdocuments/SmartDocumentsTemplatesService.kt

```diff
@@ -119,7 +119,7 @@ class SmartDocumentsTemplatesService @Inject constructor(
                                 builder.and(
                                     builder.equal(
                                         root.get<ZaakafhandelParameters>(
-                                            SmartDocumentsTemplate::zaakafhandelParameters.name
+                                            SmartDocumentsTemplateGroup::zaakafhandelParameters.name
```

Comment:
Why was this change necessary? And/or: why did the functionality even work before this change?

Reply:
Because the resulting string is still "zaakafhandleParameters" and cause both object have the field and column with the same name.